lib/idl/idl_checkers.es6.js

```diff
@@ -65,5 +65,21 @@ module.exports = [
     logger.info('Checking top-level member names');
 
     eachKey(logger, left.members || [], right.members || [], 'name');
-  }
+  },
+  function rightMissingConstructor(logger, left, right) {
```

Comment:
I thought left was blink and right was spec. Shouldn't we be checking if left is missing a constructor?